.github/CODEOWNERS

```
@@ -4,6 +4,9 @@
 # AKMODS
 akmods/ @bsherman
 
+# Aurora
+packages/aurora @inffy @NiHaiden @ledif @RealVishy @renner0e
```

```suggestion
packages/aurora @inffy @ledif @RealVishy @renner0e
```
I think CODEOWNERS is getting mad cuz of this